routing/router.go

```diff
@@ -1117,6 +1129,14 @@ func (r *ChannelRouter) networkHandler() {
 				log.Errorf("Unable to prune zombies: %v", err)
 			}
 
+		// If we are running with assumechanvalid we'll prune the graph
+		// before the prune ticker ticks for the first time.
+		case <-firstTimePrune:
```

Comment:
What about using a `time.AfterFunc` here as otherwise this case in the select is only ticks once and adds some noise to the main statement?

Reply:
> case in the select is only ticks once

Isn't the intention here to only tick once here because `graphPruneTicker` will takeover for us after the initial prune?

Reply:
Yeah, it is just to to the exact same thing as the `graphPruneTicker`, but only once. I figured this was the cleanest way of achieving it. Cleaned it up somewhat by extracting the 30 second into a constant.

Reply:
> Isn't the intention here to only tick once here because graphPruneTicker will takeover for us after the initial prune?

Yeah a `time.AfterFunc` would only tick once. By susing `firstTimePrune` we have a select statement that'll only ever execute _once_, but then adds some line noise and review burden as it makes the case statement larger.

Reply:
Alternatively, it could re-use the time channel in the `graphPruneTicker` and send a tick on that (assuming it isn't a readonly channel).

Reply:
So this would just be: 

```go

if r.cfg.AssumeChannelValid &&
	firstPruneDelay < r.cfg.GraphPruneInterval {

        time.AfterFunc(firstPruneDelay, func() {
               log.Info("Initial zombie prune starting")
                r.pruneZombieChans()
        })
}
```

Which is also a smaller diff, less code overall.

Reply:
Yeah, that was much cleaner 👌